lib/features/reps_core/reps_admin/reps_admin.info

```diff
@@ -1,15 +1,15 @@
 name = REPS admin
 description = The administration tools for the REPS application
 core = 7.x
-php = 5.6
 package = REPS
 dependencies[] = reps_core
 configure = admin/config/reps
 features[context][] = reps-admin
 features[ctools][] = context:context:3
 features[ctools][] = strongarm:strongarm:1
+features[ctools][] = views:views_default:3.0
```

Comment:
I don't see views added into this PR. Then why this line?
Perhaps there is already a view in this feature but this line was missing?

Reply:
Exactly the line was missing because we view is used since the beginning.
I've reexported the feature so the missing line has been added